packages/inter-protocol/test/price/test-fluxAggregator.js

```diff
@@ -703,3 +683,33 @@ test('storage keys', async t => {
     'mockChainStorageRoot.priceAggregator.LINK-USD_price_feed',
   );
 });
+
+test('disabling', async t => {
+  const { creatorFacet, manualTimer } = await t.context.makeChainlinkAggregator(
+    defaultConfig,
+  );
+
+  const kitA = await creatorFacet.initOracle('agoric1priceOracleA');
+  const kitB = await creatorFacet.initOracle('agoric1priceOracleB');
+  const kitC = await creatorFacet.initOracle('agoric1priceOracleC');
+
+  // ----- pushes drive a price
+  await manualTimer.tick();
+  await E(kitA.oracle).pushPrice({ roundId: 1, unitPrice: 100n });
+  await E(kitB.oracle).pushPrice({ roundId: 1, unitPrice: 200n });
+  await E(kitC.oracle).pushPrice({ roundId: 1, unitPrice: 300n });
+  await manualTimer.tick();
+
+  const round1Attempt1 = await E(creatorFacet).getRoundData(1);
+  t.is(round1Attempt1.roundId, 1n);
+  t.is(round1Attempt1.answer, 200n);
+
+  // now disable one
+  await E(kitA.admin).disable();
+  await t.throwsAsync(
+    E(kitA.oracle).pushPrice({ roundId: 2, unitPrice: 100n }),
+    {
+      message: 'pushPrice for disabled oracle',
+    },
+  );
+});
```

Comment:
We should test that aggregation continues to work.

What's the minimum number of sources that are required in order to calculate an aggregation? Does the disabled source count toward that total?

Reply:
> We should test that aggregation continues to work.

Ok, will do.

> What's the minimum number of sources that are required in order to calculate an aggregation?

It's specified in `minSubmissionCount` of the config when creating a feed. It gets read here,
https://github.com/Agoric/agoric-sdk/blob/adc0836e01eab0fd6084ecf3f59687668a98ed54/packages/inter-protocol/src/price/roundsManager.js#L363-L367

> Does the disabled source count toward that total?

It doesn't change anything. It's just as if the oracle wallet never pushed a price.